core/src/main/java/it/pagopa/selfcare/onboarding/core/InstitutionServiceImpl.java

```diff
@@ -168,6 +174,14 @@ public void onboardingProduct(OnboardingData onboardingData) {
         log.trace("onboarding end");
     }
 
+    private void checkProductAvaibility(String productId, InstitutionType institutionType, String externalInstitutionId) {
+        if(InstitutionType.PT == institutionType && VALID_PT_PRODUCTS.stream().noneMatch(s -> s.equalsIgnoreCase(productId))){
+            throw new OnboardingNotAllowedException(String.format(ONBOARDING_NOT_ALLOWED_ERROR_MESSAGE_TEMPLATE,
+                    externalInstitutionId,
+                    productId));
+        }
+    }
+
```

Comment:
avaibility will be retrived by product's dedicated status....we should introduce a new state to indicate is a PT can be assigned to that product....